Automate build on netlify

relacsua opened the pull request:
Fixes #1 

> Is there a way to automate build 24 hours?

Yes. After some research, I end by going with [Github Actions](https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) to schedule daily builds. Other solutions use other 3rd party apps like zapier, easycron, etc.

> Effects of building on high frequency

The shortest duration we can go is `Every 5 minutes`, and I've currently set to build daily at midnight. At 5 minutes, it works well for a [sample app](https://upbeat-jennings-ca115b.netlify.com/) (with just two pages) I created to play with the build. You can see all the deployments [here](https://github.com/relacsua/learning/actions). As discussed previously, the bottleneck for us is the cost of making calls to Youtube API.

ruqqq commented:
Any reason for not using official action (https://github.com/netlify/actions/blob/master/build/README.md)?

relacsua commented:
> Any reason for not using official action (https://github.com/netlify/actions/blob/master/build/README.md)?

As discussed irl, triggering the build is a trivial task(making a cURL request) and most of information on build step is configured on Netlify instead of calling it via Github Action,

> @ruqqq: Another I forgot is that, any implication with committing in the build hook URL? It’s public repo _(from telegram)_

Good point, deleted previous trigger, uploaded new ones and loading them via Github secrets.

>  @tengkuhafidz: eh soryy, this means you must create a new file for your action? _(from telegram)_

Renamed file to `deploy.yml`.
